mmv1/products/pubsub/terraform.yaml

```diff
@@ -112,7 +112,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           topic_name: "example-topic"
           subscription_name: "example-subscription"
       - !ruby/object:Provider::Terraform::Examples
-        name: "pubsub_subscription_push_bq"
```

Comment:
I guess we can keep the original name. `push_bq` seems accurate. Sorry I missed the `bq` part in my first review.

Reply:
where did you see `push` for this case?